include/zephyr/posix/unistd.h

```diff
@@ -11,16 +11,212 @@
+#define _POSIX_PRIORITY_SCHEDULING	  (-1L)
+#define _POSIX_RAW_SOCKETS		  Z_SC_VAL_IFDEF(CONFIG_NET_SOCKETS_PACKET, _POSIX_VERSION)
+#define _POSIX_READER_WRITER_LOCKS	  Z_SC_VAL_IFDEF(CONFIG_PTHREAD_IPC, _POSIX_VERSION)
+#define _POSIX_REALTIME_SIGNALS		  (-1L)
+#define _POSIX_REGEXP			  (-1L)
+#define _POSIX_SAVED_IDS		  (-1L)
+#define _POSIX_SEMAPHORES		  Z_SC_VAL_IFDEF(CONFIG_PTHREAD_IPC, _POSIX_VERSION)
+#define _POSIX_SHARED_MEMORY_OBJECTS	  (-1L)
+#define _POSIX_SHELL			  (-1L)
+#define _POSIX_SPAWN			  (-1L)
+#define _POSIX_SPIN_LOCKS		  Z_SC_VAL_IFDEF(CONFIG_PTHREAD_SPINLOCK, _POSIX_VERSION)
+#define _POSIX_SPORADIC_SERVER		  (-1L)
+#define _POSIX_SYNCHRONIZED_IO		  (-1L)
+#define _POSIX_THREAD_ATTR_STACKADDR	  Z_SC_VAL_IFDEF(CONFIG_PTHREAD_IPC, _POSIX_VERSION)
+#define _POSIX_THREAD_ATTR_STACKSIZE	  Z_SC_VAL_IFDEF(CONFIG_PTHREAD_IPC, _POSIX_VERSION)
+#define _POSIX_THREAD_CPUTIME		  (-1L)
+#define _POSIX_THREAD_PRIO_INHERIT	  _POSIX_VERSION
+#define _POSIX_THREAD_PRIO_PROTECT	  (-1L)
+#define _POSIX_THREAD_PRIORITY_SCHEDULING Z_SC_VAL_IFDEF(CONFIG_PTHREAD_IPC, _POSIX_VERSION)
+#define _POSIX_THREAD_PROCESS_SHARED	  (-1L)
+#define _POSIX_THREAD_ROBUST_PRIO_INHERIT (-1L)
+#define _POSIX_THREAD_ROBUST_PRIO_PROTECT (-1L)
+#define _POSIX_THREAD_SAFE_FUNCTIONS	  Z_SC_VAL_IFDEF(CONFIG_PTHREAD_IPC, _POSIX_VERSION)
+#define _POSIX_THREAD_SPORADIC_SERVER	  (-1L)
+#ifndef _POSIX_THREADS
+	#define _POSIX_THREADS		  Z_SC_VAL_IFDEF(CONFIG_PTHREAD_IPC, _POSIX_VERSION)
+#endif
+#define _POSIX_TIMEOUTS			  Z_SC_VAL_IFDEF(CONFIG_POSIX_CLOCK, _POSIX_VERSION)
+#define _POSIX_TIMERS			  Z_SC_VAL_IFDEF(CONFIG_POSIX_CLOCK, _POSIX_VERSION)
+#define _POSIX_TRACE			  (-1L)
+#define _POSIX_TRACE_EVENT_FILTER	  (-1L)
+#define _POSIX_TRACE_INHERIT		  (-1L)
+#define _POSIX_TRACE_LOG		  (-1L)
+#define _POSIX_TYPED_MEMORY_OBJECTS	  (-1L)
+#define _POSIX_V6_ILP32_OFF32		  (-1L)
+#define _POSIX_V6_ILP32_OFFBIG		  (-1L)
+#define _POSIX_V6_LP64_OFF64		  (-1L)
+#define _POSIX_V6_LPBIG_OFFBIG		  (-1L)
+#define _POSIX_V7_ILP32_OFF32		  (-1L)
+#define _POSIX_V7_ILP32_OFFBIG		  (-1L)
+#define _POSIX_V7_LP64_OFF64		  (-1L)
+#define _POSIX_V7_LPBIG_OFFBIG		  (-1L)
+#define _POSIX2_C_BIND			  _POSIX_VERSION
+#define _POSIX2_C_DEV			  (-1L)
+#define _POSIX2_CHAR_TERM		  (-1L)
+#define _POSIX2_FORT_DEV		  (-1L)
+#define _POSIX2_FORT_RUN		  (-1L)
+#define _POSIX2_LOCALEDEF		  (-1L)
+#define _POSIX2_PBS			  (-1L)
+#define _POSIX2_PBS_ACCOUNTING		  (-1L)
+#define _POSIX2_PBS_CHECKPOINT		  (-1L)
+#define _POSIX2_PBS_LOCATE		  (-1L)
+#define _POSIX2_PBS_MESSAGE		  (-1L)
+#define _POSIX2_PBS_TRACK		  (-1L)
+#define _POSIX2_SW_DEV			  (-1L)
+#define _POSIX2_UPE			  (-1L)
+#define _XOPEN_CRYPT			  (-1L)
+#define _XOPEN_ENH_I18N			  (-1L)
+#define _XOPEN_REALTIME			  (-1L)
+#define _XOPEN_REALTIME_THREADS		  (-1L)
+#define _XOPEN_SHM			  (-1L)
+#define _XOPEN_STREAMS			  (-1L)
+#define _XOPEN_UNIX			  (-1L)
+#define _XOPEN_UUCP			  (-1L)
+
+/* Maximum values */
```

Comment:
Should this say `Minimum values`

Reply:
If you look into https://pubs.opengroup.org/onlinepubs/009695399/basedefs/limits.h.html the meaning of maximum and minimum is inverted. I was using the document as a reference for the implementation